ci_framework/roles/ci_multus/tasks/cleanup.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Clean OCP resources if available
+  when: cifmw_openshift_kubeconfig is defined
+  block:
+    - name: Fetch NADs in multus namespace
```

Comment:
OK. But if you use a dedicated namespace like suggested before remind to add a cleanup up it here.

Reply:
done